src/main/java/de/imi/mopat/controller/ClinicController.java

```diff
@@ -148,17 +173,55 @@ public String editClinic(@RequestParam(value = "id", required = false) final Lon
         List<UserDTO> assignedUserDTOs = userService.getAssignedUserDTOs(clinicId);
         List<UserDTO> availableUserDTOs = userService.getAvailableUserDTOs(clinicId);
 
-        clinicDTO.setAssignedUserDTOs(assignedUserDTOs);
+        List<ClinicConfigurationDTO> clinicConfigurationDTOS = new ArrayList<>();
```

Comment:
please divide this into multiple functions to reduce the amount of nested expressions